src/core/connection.c

```diff
@@ -2820,11 +2814,8 @@ QuicConnRecvVerNeg(
         // Check to see if this is the current version.
         //
         if (ServerVersionList[i] == Connection->Stats.QuicVersion) {
-            QuicTraceLogConnVerbose(
-                InvalidVerNeg,
-                Connection,
-                "Dropping version negotation that includes the current version");
-            goto Exit;
+            QuicPacketLogDrop(Connection, Packet, "Version Negotation that includes the current version");
```

Comment:
shouldn't this be an error?

Reply:
No. Just a packet drop. Anything bad with an unauthenticated packet is just a drop.